packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -208,14 +208,49 @@ describe('plugin-conversation', function () {
         .then((reason) => assert.equal(reason.status, 403)));
     });
 
+
     describe('#post()', () => {
       let message, richMessage;
 
       beforeEach(() => {
         message = 'mccoy, THIS IS A TEST MESSAGE';
         richMessage = `<webex-mention data-object-id="${mccoy.id}" data-object-type="person">mccoy</webex-mention>, THIS IS A TEST MESSAGE`;
       });
+      describe('when encryption key need to rotate', () => {
+        let convo;
+
+        beforeEach(() => {
+          if (convo) {
```

Comment:
This block is not needed. If the convo object is defined at this point, we have an issue with our test runner or test setup. Please remove.

Reply:
Okay. will remove